src/elementext.rs

```diff
@@ -79,14 +77,6 @@ impl ElementExt for xmltree::Element {
                 .into()
             })
     }
-
-    fn check_name(&self, name: &str) -> crate::Result<()> {
-        if self.name == name {
-            Ok(())
-        } else {
-            Err(error::WrongName::new(name, self).into())
```

Comment:
If we go with replacing the wrong name error with a warning, please also delete `error::WrongName`